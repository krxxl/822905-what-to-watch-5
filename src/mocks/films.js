export default [{
  name: `The Grand Budapest Hotel`,
  genre: `comedy`,
  year: `2010`,
  poster: `img/the-grand-budapest-hotel-poster.jpg`,
  preview: `img/bohemian-rhapsody.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `1h 39m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  isInList: true,
  id: 1,
},
{
  name: `bohemian-rhapsody`,
  genre: `Sci-Fi`,
  year: `2010`,
  poster: `img/bohemian-rhapsody.jpg`,
  preview: `img/bohemian-rhapsody.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `2h 01m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  isInList: true,
  id: 2,
}, {
  name: `dardjeeling-limited`,
  genre: `comedy`,
  year: `2010`,
  poster: `img/dardjeeling-limited.jpg`,
  preview: `img/dardjeeling-limited.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `1h 01m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  isInList: true,
  id: 3,
}, {
  name: `fantastic-beasts-the-crimes-of-grindelwald`,
  genre: `comedy`,
  year: `2010`,
  poster: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  preview: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `1h 15m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  isInList: true,
  id: 4,
}, {
  name: `johnny-english`,
  genre: `comedy`,
  year: `2010`,
  poster: `img/johnny-english.jpg`,
  preview: `img/johnny-english.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `1h 39m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  isInList: true,
  id: 5,
}, {
  name: `macbeth`,
  genre: `comedy`,
  year: `2010`,
  poster: `img/macbeth.jpg`,
  preview: `img/macbeth.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `1h 39m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  isInList: true,
  id: 6,
}, {
  name: `midnight-special`,
  genre: `kids & family`,
  year: `2010`,
  poster: `img/midnight-special.jpg`,
  preview: `img/midnight-special.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `1h 39m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  isInList: true,
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  id: 7,
}, {
  name: `moonrise-kingdom`,
  genre: `thriller`,
  year: `2010`,
  poster: `img/moonrise-kingdom.jpg`,
  preview: `img/moonrise-kingdom.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `1h 39m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  isInList: true,
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  id: 8,
}, {
  name: `moonrise-kingdom`,
  genre: `horror`,
  year: `2010`,
  poster: `img/moonrise-kingdom.jpg`,
  preview: `img/moonrise-kingdom.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `1h 39m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  isInList: true,
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  id: 9,
}, {
  name: `moonrise-kingdom`,
  genre: `documentary`,
  year: `2010`,
  poster: `img/moonrise-kingdom.jpg`,
  preview: `img/moonrise-kingdom.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `1h 39m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  isInList: true,
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  id: 10,
}, {
  name: `moonrise-kingdom`,
  genre: `drama`,
  year: `2010`,
  poster: `img/moonrise-kingdom.jpg`,
  preview: `img/moonrise-kingdom.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `1h 39m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  isInList: true,
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  id: 11,
}, {
  name: `moonrise-kingdom`,
  genre: `Sci-Fi`,
  year: `2010`,
  poster: `img/moonrise-kingdom.jpg`,
  preview: `img/moonrise-kingdom.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `1h 39m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  isInList: true,
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  id: 12,
}, {
  name: `moonrise-kingdom`,
  genre: `Romance`,
  year: `2010`,
  poster: `img/moonrise-kingdom.jpg`,
  preview: `img/moonrise-kingdom.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `1h 39m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  isInList: true,
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  id: 13,
}, {
  name: `moonrise-kingdom`,
  genre: `Kids & Family`,
  year: `2010`,
  poster: `img/moonrise-kingdom.jpg`,
  preview: `img/moonrise-kingdom.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `1h 39m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  isInList: true,
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  id: 14,
}, {
  name: `moonrise-kingdom`,
  genre: `Horror`,
  year: `2010`,
  poster: `img/moonrise-kingdom.jpg`,
  preview: `img/moonrise-kingdom.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `1h 39m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  isInList: true,
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  id: 15,
}, {
  name: `moonrise-kingdom`,
  genre: `Drama`,
  year: `2010`,
  poster: `img/moonrise-kingdom.jpg`,
  preview: `img/moonrise-kingdom.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `1h 39m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  isInList: true,
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  id: 16,
}, {
  name: `moonrise-kingdom`,
  genre: `Documentary`,
  year: `2010`,
  poster: `img/moonrise-kingdom.jpg`,
  preview: `img/moonrise-kingdom.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `1h 39m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  isInList: true,
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  id: 17,
}, {
  name: `moonrise-kingdom`,
  genre: `Crime`,
  year: `2010`,
  poster: `img/moonrise-kingdom.jpg`,
  preview: `img/moonrise-kingdom.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `1h 39m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  isInList: true,
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  id: 18,
}, {
  name: `dardjeeling-limited`,
  genre: `comedy`,
  year: `2010`,
  poster: `img/dardjeeling-limited.jpg`,
  preview: `img/dardjeeling-limited.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `1h 01m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  isInList: true,
  id: 19,
}, {
  name: `dardjeeling-limited`,
  genre: `comedy`,
  year: `2010`,
  poster: `img/dardjeeling-limited.jpg`,
  preview: `img/dardjeeling-limited.jpg`,
  hero: `img/bg-the-grand-budapest-hotel.jpg`,
  time: `1h 01m`,
  rating: `9.8`,
  count: 240,
  desc: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustaves friend and protege.  Gustave prides himself on providing first-class service to the hotels guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustaves lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  director: `Wes Andreson`,
  starring: [`Bill Murray`,
    `Edward Norton`,
    `Jude Law`,
    `Willem Dafoe`,
    `Saoirse Ronan`,
    `Tony Revoloru`],
  video: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  isInList: true,
  id: 20,
},
];
