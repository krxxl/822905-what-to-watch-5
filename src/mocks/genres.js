export const genres = [
  {
    name: `All genres`,
    id: 1
  },
  {
    name: `Comedies`,
    id: 2
  },
  {
    name: `Crime`,
    id: 3
  },
  {
    name: `Documentary`,
    id: 4
  },
  {
    name: `Dramas`,
    id: 5
  },
  {
    name: `Horror`,
    id: 6
  },
  {
    name: `Kids & Family`,
    id: 7
  },
  {
    name: `Romance`,
    id: 8
  },
  {
    name: `Sci-Fi`,
    id: 9
  },
  {
    name: `Thrillers`,
    id: 10
  },
];

export const genresInSingular = {
  ALL: `All genres`,
  COMEDY: `Comedies`,
  CRIME: `Crime`,
  DOCUMENTARY: `Documentary`,
  DRAMA: `Dramas`,
  HORROR: `Horror`,
  KIDS: `Kids & Family`,
  ROMANCE: `Romance`,
  SCI_FI: `Sci-Fi`,
  THRILLER: `Thrillers`,
};
