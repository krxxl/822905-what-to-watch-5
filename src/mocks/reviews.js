export default [
  {
    id: 1,
    reviews: [
      {
        text: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        author: `Matthew Lickona`,
        date: `December 24, 2016`,
        rating: 8.9,
        id: 1
      },
      {
        text: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        author: `Paula Fleri-Soler`,
        date: `December 20, 2016`,
        rating: 8.9,
        id: 2
      },
      {
        text: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        author: `Bill Goodykoontz`,
        date: `November 18, 2015`,
        rating: 8.9,
        id: 3
      },
    ]
  },
  {
    id: 2,
    reviews: [
      {
        text: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        author: `Matthew Lickona`,
        date: `December 20, 2016`,
        rating: 8.9,
        id: 4
      },
      {
        text: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        author: `Bill Goodykoontz`,
        date: `December 24, 2016`,
        rating: 8.9,
        id: 5
      },
    ]
  },
  {
    id: 3,
    reviews: [
      {
        text: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        author: `Matthew Lickona`,
        date: `December 24, 2016`,
        rating: 8.9,
        id: 6
      },
    ]
  },
  {
    id: 4,
    reviews: [
      {
        text: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        author: `Matthew Lickona`,
        date: `November 18, 2015`,
        rating: 8.9,
        id: 7
      },
      {
        text: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        author: `Paula Fleri-Soler`,
        date: `December 24, 2016`,
        rating: 8.9,
        id: 8
      },
      {
        text: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        author: `Bill Goodykoontz`,
        date: `December 24, 2016`,
        rating: 8.9,
        id: 9
      },
    ]
  },
  {
    id: 5,
    reviews: [
      {
        text: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        author: `Matthew Lickona`,
        date: `December 24, 2016`,
        rating: 8.9,
        id: 10
      },
      {
        text: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        author: `Paula Fleri-Soler`,
        date: `December 20, 2016`,
        rating: 8.9,
        id: 11
      },
    ]
  },
  {
    id: 6,
    reviews: [
      {
        text: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        author: `Matthew Lickona`,
        date: `December 24, 2016`,
        rating: 8.9,
        id: 12
      },
    ]
  },
  {
    id: 7,
    reviews: [
      {
        text: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        author: `Matthew Lickona`,
        date: `December 24, 2016`,
        rating: 8.9,
        id: 13
      },
      {
        text: `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`,
        author: `Paula Fleri-Soler`,
        date: `December 20, 2016`,
        rating: 8.9,
        id: 14
      },
      {
        text: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        author: `Bill Goodykoon`,
        date: `November 18, 2015`,
        rating: 8.9,
        id: 15
      },
    ]
  },
  {
    id: 8,
    reviews: [
      {
        text: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
        author: `Matthew Lickona`,
        date: `December 24, 2016`,
        rating: 8.9,
        id: 16
      },
    ]
  }
];
