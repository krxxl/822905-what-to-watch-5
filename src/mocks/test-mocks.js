import films from './films';
import reviews from './reviews';
// import genres from './genres';
import {AuthorizationStatus} from "../constant/constant";
import {SHOW_ON_STAR_FILMS} from '../constant/constant';

export const TEST_MOCK_STORE = {
  DATA: {
    films: films,
    reviews: reviews,
    favoriteFilms: films,
    isLoading: false,
    isPromoLoading: false, 
    sendingReview: false,
    sendingReviewError: false,
    promoFilm: films[0],
  },
  USER: {
    authorizationStatus: AuthorizationStatus.NO_AUTH,
  },
  SHOW: {
    genreActive: `All genres`,
    count: SHOW_ON_STAR_FILMS, 
  }
};

