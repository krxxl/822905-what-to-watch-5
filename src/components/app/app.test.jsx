import React from 'react';
import renderer from 'react-test-renderer';
import App from './app';
import {Provider} from 'react-redux';
import configureMockStore from 'redux-mock-store';
import {TEST_MOCK_STORE} from '../../mocks/test-mocks';
import thunk from 'redux-thunk';
import {createAPI} from '../../services/api';

const api = createAPI(() => {});

const middlewares = [thunk.withExtraArgument(api)];
const mockStore = configureMockStore(middlewares);
const store = mockStore(TEST_MOCK_STORE);

describe(`Render App`, () => {
  it(`Render App`, () => {
    const tree = renderer
      .create(
          <Provider store={store}>
            <App />
          </Provider>,
          {
            createNodeMock: () => {
              return {};
            },
          }
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
