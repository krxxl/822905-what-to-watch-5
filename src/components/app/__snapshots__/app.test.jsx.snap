// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render App Render App 1`] = `
<h1>
  LOADING...
</h1>
`;
